### PR TITLE
Adding the compile time to the watcher for development

### LINKED ISSUE #1851: Add the compile time to the watcher in development
I like seeing how long my assets take to build. Would be neat to see how long the compilation takes as well.

Maybe our "Done" message could just mimic the one in green here?

image

### PR DESCRIPTION
**Pull request by jwoertink**
## Purpose
Fixes #1851

## Description

This adds in the time span of compiling your app when it boots. 

Here's what it looks like
image

## Checklist
* [x] - An issue already exists detailing the issue/or feature request that this PR fixes
* [x] - All specs are formatted with `crystal tool format spec src`
* [x] - Inline documentation has been added and/or updated
* [x] - Lucky builds on docker with `./script/setup`
* [x] - All builds and specs pass on docker with `./script/test`

### CODE REVIEW

File: tasks/watch.cr [hunk 1]
```diff
@@ -262,7 +264,14 @@ module LuckySentry
       if build_success
         self.app_built = true
         create_app_processes
-        puts "#{" Done ".colorize.on_cyan.black} compiling"
+
+        elapsed_time = Time.monotonic - @build_started
+        message = String.build do |io|
+          io << " DONE ".colorize.on_cyan.black
+          io << " Compiled successfully in #{elapsed_time}".colorize.cyan
```
Comment (robacarp) [inline tasks/watch.cr:271]: What about calling distance_of_time_in_words or something on this so it's a little easier on the eyes?

**Review by robacarp — APPROVED** [review]

File: tasks/watch.cr (hunk 1 above)
Comment (jwoertink) [reply tasks/watch.cr:271]: oh, that's a good idea. I was thinking I didn't want to write out a whole deal, but I forgot I could just use that.

Comment (jwoertink) [reply tasks/watch.cr]: Alright, made these changes and updated the screenshot. It looks a lot better.

Comment (robacarp) [reply tasks/watch.cr]: excellent!

**Pull request merged**